uint32 index
string name
geometry_msgs/Point position
